### PR TITLE
Reduce yaml schedule files for minimal+role_minimal testsuites

### PR DESCRIPTION
**Pull request by chcao**
Reduce yaml schedule files for minimal+role_minimal testsuites. For
s390x kvm, the installation is done via VNC, then Graphical target is
selected independently on the role. So we load verify_default_target
instead of validate_default_target, and on s390x test_data, we set
default target as graphical.target.

- Related ticket: https://progress.opensuse.org/issues/163256
- Needles: n/a
- Verification run: 
- https://openqa.suse.de/tests/14999182  (aarch64)
- https://openqa.suse.de/tests/14999181  (ppc64le)
- https://openqa.suse.de/tests/14999392  (s390x)
- https://openqa.suse.de/tests/14999180  (x86_64)
- Related mr: https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/287

### CODE REVIEW

**Comment by chcao** [comment]
Checked some PRs, the "installation/installation_settings/validate_default_target" was skipped, so I remove it in this PR, too.
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19666/files

**Comment by lemon-suse** [comment]
In fact, I think use verify_default_target instead of validate_default_target is reasonable for s390 test, just please add the reason of that in the PR so that we can search PR to know the history and background.

**Comment by chcao** [comment]
> In fact, I think use verify_default_target instead of validate_default_target is reasonable for s390 test, just please add the reason of that in the PR so that we can search PR to know the history and background.

Yes, I updated the PR description, thanks.

File: schedule/yam/test_cases/minimal+role_minimal.yaml [hunk 1]
```diff
@@ -4,10 +4,6 @@ description: >
   SLE 15 specific registered installation with minimal role selected.
   Installation is validated by successful boot and that YaST does not report
   any issue.
-vars:
```
Comment (lemon-suse) [inline schedule/yam/test_cases/minimal+role_minimal.yaml:7]: Please check and add these removed vars in yaml file to corresponding test suites.

Comment (chcao) [reply schedule/yam/test_cases/minimal+role_minimal.yaml:7]: Already done, added them in MR, and added the related MR link to the description.

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**